<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuronParametersForm</class>
 <widget class="QWidget" name="NeuronParametersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <widget class="QPushButton" name="ApplyNowButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>379</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply now</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RealTimeApplyParametersCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>382</y>
     <width>140</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Real Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="ElectrodeSensitivityLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Electrode sensitivity:</string>
   </property>
  </widget>
  <widget class="QSlider" name="ElectrodeSensitivitySlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>240</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="StimCurrentSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>55</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StimCurrentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stimulation current:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoizeStimCurrentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>87</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Noize stimulation current:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NoizeStimCurrentSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>82</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SysStepLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>114</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Synaptic step:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SynStepSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>109</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="UThresholdLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>141</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>U threshold</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="UThresholdSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>136</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="VThresholdLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>168</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>V threshold:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="VThresholdSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>163</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StabilityLimitLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stability limit:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="StabilityLimitSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StimRelaxTimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>222</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>U relax time:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="StimRelaxTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>217</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StabRelaxTimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>249</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stability relax time</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="StabRelaxTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>244</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ResistanceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>276</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistance:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ResistanceSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>271</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RelaxedStimLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>303</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Relaxed U potential:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RelaxedStimSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>298</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ResMaxLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Membrane resource max:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ResMaxSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>325</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ResLimLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>357</y>
     <width>160</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Membrane resource limit:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ResMinSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>352</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
